packages/commerce-sdk-react/src/test-utils.tsx

```diff
@@ -5,39 +5,42 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import {render, RenderOptions} from '@testing-library/react'
-import jwt from 'jsonwebtoken'
-import nock from 'nock'
 import React from 'react'
-import CommerceApiProvider, {CommerceApiProviderProps} from './provider'
+import {render, RenderOptions} from '@testing-library/react'
+import {renderHook} from '@testing-library/react-hooks/dom'
 import {QueryClient, QueryClientProvider} from '@tanstack/react-query'
-import {url} from 'inspector'
+import CommerceApiProvider, {CommerceApiProviderProps} from './provider'
+
+// Note: this host does NOT exist
+// it is intentional b/c we can catch those unintercepted requests
+// from log easily. You should always make sure all requests are nocked.
+export const DEFAULT_TEST_HOST = 'http://localhost:8888'
 
-export const TEST_CONFIG = {
-    proxy: 'http://localhost:3000/mobify/proxy/api',
-    clientId: 'c9c45bfd-0ed3-4aa2-9971-40f88962b836',
-    organizationId: 'f_ecom_zzrf_001',
-    shortCode: '8o7m175y',
-    redirectURI: 'http://localhost:3000/callback',
+export const DEFAULT_TEST_CONFIG = {
+    proxy: `${DEFAULT_TEST_HOST}/mobify/proxy/api`,
+    redirectURI: `${DEFAULT_TEST_HOST}/callback`,
+    clientId: '12345678-1234-1234-1234-123412341234',
+    organizationId: 'f_ecom_zzrmy_orgf_001',
+    shortCode: '12345678',
     siteId: 'RefArchGlobal',
     locale: 'en-US',
     currency: 'USD'
 }
 
-export const queryClient = new QueryClient({
-    // During testing, we want things to fail immediately
-    defaultOptions: {queries: {retry: false}, mutations: {retry: false}}
-})
+export const createQueryClient = () => {
+    return new QueryClient({
+        // During testing, we want things to fail immediately
+        defaultOptions: {queries: {retry: false}, mutations: {retry: false}}
+    })
+}
 
-const TestProviders = (props: {
-    children: React.ReactNode
-    commerceApiProvider?: Partial<CommerceApiProviderProps>
-}) => {
-    const {commerceApiProvider} = props
+type TestProviderProps = Partial<CommerceApiProviderProps & {queryClient: QueryClient}>
 
+const TestProviders = (props: TestProviderProps) => {
+    const queryClient = props.queryClient || createQueryClient()
```

Comment:
updated `TestProvider` to accept queryClient as a prop. This enables individual unit tests to gain access to the query client as well as tearing the client down for each test.